string label
string type
geometry_msgs/Pose pose